src/components/ButtonPanel/ButtonPanel.jsx

```diff
@@ -7,32 +7,32 @@ import './ButtonPanel.scss';
 const ButtonPanel = () => (
   <div className="button-panel">
     <div className="group">
-      <Button name="AC" />
-      <Button name="+/-" />
-      <Button name="%" />
-      <Button name="÷" />
+      <Button name="AC" color="lightgrey" />
+      <Button name="+/-" color="lightgrey" />
+      <Button name="%" color="lightgrey" />
+      <Button name="/" />
```

Comment:
I trust you can do this 👇  before merging.
Kindly make use of this well-known division sign `÷` on a calculator.

If you have doubts, kindly check the requirement the previous reviewer shared with you.
Click on the `calculator` link